### ISSUE TITLE
Move issue templates into the .github folder

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-orga-app#1146]
Please don't keep the templates in the usual code area.

**Comment by srv-twry** [comment]
@iamareebjamal can you please add the templates from the settings tab for the repository ? We already have it on the server. It needs owner access hence I can't.

**Comment by iamareebjamal** [comment]
@srv-twry It is failing right now. I've contacted github support. Meanwhile, you follow the process to move the templates in .github folder

**Comment by srv-twry** [comment]
@iamareebjamal I don't think it is possible. https://blog.github.com/2018-01-25-multiple-issue-and-pull-request-templates/
We need an ISSUE_TEMPLATE directory in the root of the project for this to work. I guess we can wait for github support to fix it and then simply use that one.

**Comment by iamareebjamal** [comment]
OK
